test/zombiefactory.js

```diff
@@ -0,0 +1,37 @@
+var ZombieFactory = artifacts.require("ZombieFactory");
+
+contract("ZombieFactory", function(accounts) {
+
+	const account_one = accounts[0];
+	const account_two = accounts[1];
+	const name = "Name"
+	let contract;
+
+	beforeEach(async () => {
+        contract = await ZombieFactory.deployed();
+	});
+
+	it("extends the zombies array by exactly one with createRandomZombie", async () =>  {
+		let preLength = (await contract.getNumberOfZombies.call()).toNumber()
```

Comment:
The need to be const as it is never reassigned

Reply:
also what are the brackets around the await for?

Reply:
Brackets are for the .toNumber() call